document/badges/badges.css

```diff
@@ -0,0 +1,48 @@
+.container {
+  width: 90%;
+  margin: 0 auto;
+}
+
+.container-fluid {
+  padding: 8px;
+  box-shadow: 0px 0px 15px 0px var(--primary-theme-light-color);
+}
+
+.grid-container {
+  display: grid;
+  grid-template-rows: auto 1fr;
+  grid-template-areas:
+    "header header header"
+    "side badges badges";
```

Comment:
Can be written
header header
  side    badges

Reply:
I have divided the whole page in 3 column , give side 1/3 and badges 2/3

Reply:
it can be divided  into 2 columns
1 with less space and second with 1fr

Reply:
But then I have to give width to side section

Reply:
yes you can have more control over it

Reply:
yeah but I want that it takes width automatically with respect to content in that section